lib/OpenQA/Script/CloneJob.pm

```diff
@@ -241,10 +241,37 @@ sub clone_jobs ($jobid, $options) {
     handle_tx($tx, $url_handler, $options, $jobs) if $tx;
 }
 
+sub detect_maintaince_update($job) {
+    my %settings = %{$job->{settings}};
```

Comment:
Making a copy of the hash seems useless.

```suggestion
    my $settings = $job->{settings};
```
And then use it via `$settings->{…}`.

Reply:
Modified the code as suggested. Thanks :)